### PR TITLE
Resolved to-be-deprecated UI reference.

### PR DESCRIPTION
**Pull request by Joseph-Garvey**
Resolved soon to be deprecated edge_color reference and replaced with the new border_color following warning from terminal output.

### CODE REVIEW

**Comment by Joseph-Garvey** [comment]
Need to redo this fix and implement the pre-commit hook changes separately.

**Comment by palvarezc** [comment]
> Need to redo this fix and implement the pre-commit hook changes separately. Thought it had worked when first tested but napari-env hadn't updated the plugin with new commits.


` pip install -e` probably helps

**Comment by Joseph-Garvey** [comment]
@samcunliffe Is there any reason V3.0 of pre-commit hooks is used?

**Comment by samcunliffe** [comment]
> @samcunliffe Is there any reason V3.0 of pre-commit hooks is used?

It's 4.4 no?

https://github.com/palvarezc/cavendish-particle-tracks/blob/f902d384ebfa1526db9bf56c144ddb5679b94b01/.pre-commit-config.yaml#L3

But in any case: no reason not to update to [the latest](https://github.com/pre-commit/pre-commit-hooks/releases/tag/v4.6.0).

**Comment by Joseph-Garvey** [comment]
> > @samcunliffe Is there any reason V3.0 of pre-commit hooks is used?
> 
> It's 4.4 no?
> 
> https://github.com/palvarezc/cavendish-particle-tracks/blob/f902d384ebfa1526db9bf56c144ddb5679b94b01/.pre-commit-config.yaml#L3
> 
> But in any case: no reason not to update to [the latest](https://github.com/pre-commit/pre-commit-hooks/releases/tag/v4.6.0).

Ah, got mixed up with:
> https://github.com/palvarezc/cavendish-particle-tracks/blob/ace15daa8424c2a67af1b7ed1fbd91d0f11d12d5/.github/workflows/test_and_deploy.yml#L22

Will have to find out why that extra pre-commit I added is failing.

**Comment by samcunliffe** [comment]
> > > @samcunliffe Is there any reason V3.0 of pre-commit hooks is used?
> > 
> > 
> > It's 4.4 no?
> > https://github.com/palvarezc/cavendish-particle-tracks/blob/f902d384ebfa1526db9bf56c144ddb5679b94b01/.pre-commit-config.yaml#L3
> > 
> > But in any case: no reason not to update to [the latest](https://github.com/pre-commit/pre-commit-hooks/releases/tag/v4.6.0).
> 
> Ah, got mixed up with:
> 
> > https://github.com/palvarezc/cavendish-particle-tracks/blob/ace15daa8424c2a67af1b7ed1fbd91d0f11d12d5/.github/workflows/test_and_deploy.yml#L22
> 
> Will have to find out why that extra pre-commit I added is failing.

Ah. I did some digging. It looks like `check-illegal-windows-names` isn't in a release yet (the latest is v4.6.0 from 6th April) and [this closed bug](https://github.com/pre-commit/pre-commit-hooks/issues/1043) is dated 16th April. 

Now, why on earth it's written into the readme but not available yet... I've no idea.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -3,23 +3,32 @@ repos:
     rev: v4.4.0
     hooks:
       - id: check-docstring-first
+        description: Checks that code is placed after the docstring
+      - id: check-illegal-windows-names
+        description: Prevents the use of filenames that are incompatible with Windows
```
Comment (samcunliffe) [inline .pre-commit-config.yaml:8]:
```suggestion
      # TODO: uncomment the following when pre-commit-hooks v4.7.0 is released!
      # - id: check-illegal-windows-names
      #   description: Prevents the use of filenames that are incompatible with Windows
```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -3,23 +3,32 @@ repos:
     rev: v4.4.0
```
Comment (samcunliffe) [inline .pre-commit-config.yaml:3]:
```suggestion
    rev: v4.6.0
```
... probably doesn't hurt.

**Comment by Joseph-Garvey** [comment]
@samcunliffe So if I'm understanding you correctly, it's there, but was added after the latest packaged release?
Believe this branch is ready to merge now.

**Review by samcunliffe — APPROVED** [review]

**Comment by samcunliffe** [comment]
> @samcunliffe So if I'm understanding you correctly, it's there, but was added after the latest packaged release? 

Yeah exactly.

**Pull request merged**